java/src/apps/GuiLafConfigPane.java

```diff
@@ -97,7 +97,7 @@ void doClickSelection(JPanel panel) {
     void doVerticalToolBar(JPanel panel) {
         panel.setLayout(new FlowLayout());
         mouseEvent = new JCheckBox(ConfigBundle.getMessage("GUIButtonVerticalToolBar"));
-        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isVerticalToolBar());
+        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isToolBarVertical());
```

Comment:
FYI: The "Toolbar Side" (top, left, bottom, right) UserPreferences code in the LayoutEditor is per-panel.